src/core/issues-notes.js

```diff
@@ -203,6 +193,30 @@ function createIssueSummaryEntry(l10nIssue, report, id) {
   `;
 }
 
+/**
+ *
+ * @param {*} issueList
+ */
+function makeIssueSectionSummary(issueList) {
+  const issueSummaryElement = document.getElementById("issue-summary");
+
+  if (!issueSummaryElement) return;
+  const heading = issueSummaryElement.querySelectorAll("h2, h3, h4, h5, h6");
+  const defaultHeading = hyperHTML`<h2>${l10n.issue_summary}</h2>`;
+
+  issueList.hasChildNodes()
+    ? issueSummaryElement.append(issueList)
```

Comment:
I'm a bit worried that this `issueSummaryElement.append(issueList)` appears twice (here and above)? I was only expecting to see that once.

Reply:
I noticed that and tested it for various cases and it works fine. It does not duplicate the content. Should I do it another way?

Reply:
yeah, please see if there is a way that `issueSummaryElement.append(issueList)` is only ever called once.